### ISSUE TITLE
Bookwyrm.social reviews not showing(or showing partly) on another instances

### ISSUE THREAD

**Issue by tali-vitali** [bookwyrm-social/bookwyrm#2790]
**Describe the bug**
Bookwyrm.social reviews not showing(or showing partly) on another instances

**To Reproduce**
Example 1:
"Fairy Tail" by Stephen King
https://bookwyrm.social/book/429411/s/fairy-tale (12 reviews)
https://orreadi.com/book/14739/s/fairy-tale (3 reviews)
https://buecher.pnpde.social/book/5051/s/fairy-tale (2 reviews)

Example 2:
"The Ballad of Songbirds and Snakes" by Suzanne Collins
https://bookwyrm.social/book/107589/s/the-ballad-of-songbirds-and-snakes (18 reviews)
https://orreadi.com/book/32174/s/the-ballad-of-songbirds-and-snakes (1 review)
https://buecher.pnpde.social/book/7399/s/the-ballad-of-songbirds-and-snakes (1 review)

**Expected behavior**
Same reviews on each example.

**Instance**
orreadi.com(0.6.0 bookwyrm version), buecher.pnpde.social(0.6.1 bookwyrm version)

**Comment by WesleyAC** [comment]
I think this is largely expected behaviour, given how ActivityPub works. Reviews are only federated to instances when someone on that instance follows the person who published the review. This is similar to how things work in other ActivityPub projects — on Mastodon, for instance, posts are only federated where there's a follow relationship.

There are ways to improve this — for instance, we could add support for ActiivtyPub relays — but on the whole, this can be a touchy subject, and it's one that there's been a lot of discussion about and various solutions proposed in the broader ActivityPub ecosystem.

**Comment by tali-vitali** [comment]
@WesleyAC,
  I think maximum number of reviews very important in all instances for project like bookwyrm(mb unlike other fediverse projects), at least for book view(e.g. https://bookwyrm.social/book/429411/s/fairy-tale). Important for discovering what to read next.
  Feed good(and should be) without reviews - understandable.
  Otherwise why new common reader(not researcher, not native of other language then english) will choose other instances with less users and correspondingly less reviews. And then use chosen instance for social activity and both chosen instance and boowyrm.social for discovering what to read.

**Comment by WesleyAC** [comment]
I agree that this is a problem for users that is worth improving. I wanted to be clear that I don't expect to ever have 100% coverage, and I don't think that should be the goal, but making it easier for instances, especially small instances to share reviews seems good.

In terms of practical solutions, enabling the use of relays for this seems like the most impactful thing, so I'll open a issue for that, but I'm happy to hear other suggestions as well.

**Comment by WesleyAC** [comment]
Created #2802 for discussion of relays specifically as a solution to this problem.

**Comment by WesleyAC** [comment]
#2005 is also relevant to this problem

**Comment by LokiMidgard** [comment]
There could also be an option to temporarily load and list reviews from other instances. Similar to search, where you can search other catalogs and it will sometimes list dozens of wyrmbook instances.
